packages/chakra-ui/src/range-field/RangeField.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useCallback } from "react";
+import { RangeField } from "./RangeField";
+import { fieldAtom, formAtom, useForm } from "form-atoms";
+import { StoryForm } from "../stories";
+import { zodValidate } from "form-atoms/zod";
+import { z } from "zod";
+import { NumberFieldAtom } from "@react-last-field/field/src";
+
+export default {
+  title: "RangeField",
+  component: RangeField,
+};
+
+const form = formAtom({
+  rating: fieldAtom({
+    value: undefined,
+    validate: zodValidate(z.number().min(0).max(20)),
+  }) as NumberFieldAtom, // TODO: Improve typing
+});
+
+const Template = () => {
+  const { fieldAtoms, submit } = useForm(form);
+
+  return (
+    <StoryForm submit={submit}>
+      <RangeField
+        field={fieldAtoms.rating}
+        defaultValue={3} // TODO: Field atom value should be set to defaultValue when defined
```

Comment:
```suggestion
        defaultValue={3} // TODO: Field atom value should be set to defaultValue when defined
```
there is hook for this behavior
https://github.com/jaredLunde/form-atoms#usefieldinitialvalue

maybe good idea to support this.

Previously I intented to set initial values only via the `fieldAtom` constructors!

But maybe there will be usefull cases to do it from the component side!

Reply:
Would be great if it would be supported by the package! Should be solved: [7d1c68d](https://github.com/MiroslavPetrik/form-atoms-field/pull/8/commits/7d1c68d1cf89dfe132debdf2960c4030647ff626)